Road/endpoints/endpoints.py

```diff
@@ -10,8 +11,12 @@ def _health_check(request: Request) -> dict:
     response = {
         "message": HTTPStatus.OK.phrase,
         "status-code": HTTPStatus.OK,
-        "data": {}
+        "data": {},
     }
+    print(read_n_from_file())
```

Comment:
Please save the returned value instead of print the result

Reply:
this line should be deleted , it was for testing only